rarible/networks.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mainnet": {
+    "ExchangeV1": {
+      "address": "0xcd4ec7b66fbc029c116ba9ffb3e59351c20b5b06"
```

Comment:
This file should all V1, V2 & V3 address?

Reply:
Just need to add V2 address right? V3 is implemented as upgrade or V2 only